resources/views/checkout/partials/sections/login/logged-out.blade.php

```diff
@@ -1,31 +1,73 @@
+            @lang('We will send your order confirmation to this e-mail address. We will also check if you already have an account so you can checkout more efficiently.')
+        </p>
 
-<p v-if="!emailAvailable" class="self-end text-ct-inactive">
-    @lang('You already have an account with this e-mail address. Please log in to continue.')
-</p>
-<p v-else class="self-end text-ct-inactive">
-    @lang('We will send your order confirmation to this e-mail address. We will also check if you already have an account so you can checkout more efficiently.')
-</p>
-<x-rapidez-ct::button.primary v-if="!emailAvailable" v-on:click.prevent="go" dusk="continue">
-    @lang('Login')
-</x-rapidez-ct::button.primary>
+        <template v-if="!loggedIn && (!checkoutLogin.isEmailAvailable || checkoutLogin.createAccount)">
+            <x-rapidez-ct::input.password
+                label="Password"
+                name="password"
+                v-model="checkoutLogin.password"
+                required
+            />
+        </template>
+        <p v-if="!loggedIn && !checkoutLogin.isEmailAvailable" class="self-end text-ct-inactive">
+            @lang('You already have an account with this e-mail address. Please log in to continue.')
+            <a href="{{ route('account.forgotpassword') }}" class="underline hover:no-underline">@lang('Forgot your password?')</a>
+        </p>
+        @if (App::providerIsLoaded('Rapidez\Account\AccountServiceProvider'))
+            <a href="{{ route('account.forgotpassword') }}" class="inline-block text-sm hover:underline mt-5" v-if="!checkoutLogin.isEmailAvailable">
+                @lang('Forgot your password?')
+            </a>
+        @endif
+        <template v-if="!loggedIn && checkoutLogin.createAccount">
+            <x-rapidez-ct::input.password
+                label="Repeat password"
+                name="password_repeat"
+                v-model="checkoutLogin.password_repeat"
+                required
+            />
+            <x-rapidez-ct::input
+                label="Firstname"
+                name="firstname"
+                type="text"
+                v-model="checkoutLogin.firstname"
+                required
+            />
+            <x-rapidez-ct::input
+                label="Lastname"
+                name="lastname"
+                type="text"
+                v-model="checkoutLogin.lastname"
+                required
+            />
+        </template>
+
+        <template v-if="!loggedIn && checkoutLogin.isEmailAvailable">
+            <div class="col-span-full">
+                <x-rapidez-ct::input.checkbox v-model="checkoutLogin.createAccount" dusk="create_account">
+                    @lang('Create an account')
+                </x-rapidez-ct::input.checkbox>
+            </div>
+        </template>
+
+        <x-rapidez-ct::button.accent type="button" v-on:click="checkoutLogin.go" v-if="checkoutLogin.isEmailAvailable && checkoutLogin.createAccount" dusk="register">
```

Comment:
Do we need `type="button"` here?

Reply:
Yes, because they are wrapped in a <form, this prevents it from submitting